src/java.base/share/classes/java/nio/channels/FileChannel.java

```diff
@@ -270,6 +270,11 @@ protected FileChannel() { }
      *          support creating file channels, or an unsupported open option is
      *          specified, or the array contains an attribute that cannot be set
      *          atomically when creating the file
+     * @throws  FileAlreadyExistsException
+     *          if a file of that name already exists and the {@link
+     *          StandardOpenOption#CREATE_NEW CREATE_NEW} option is specified
+     *          and the file is being opened for writing
+     *          <i>(optional specific exception)</i>
```

Comment:
This may be the first usage of optional specific exceptions in the java.nio.channels API docs. It's okay to do this but it will require linking to the java.nio.file package description where this is defined. You'll also need to import the exception or use the fully qualified class file here. Probably best to replace "if" with "If" to be consistent with the other exception messages here.

Reply:
Thanks for adding the `nio` label. I had mentioned the package description already and will update that. There is not much consistency in whether the first letter of the `@throws` verbiage is upper case.